Formula/libjson-rpc-cpp.rb

```diff
@@ -1,8 +1,8 @@
 class LibjsonRpcCpp < Formula
   desc "C++ framework for json-rpc"
   homepage "https://github.com/cinemast/libjson-rpc-cpp"
-  url "https://github.com/cinemast/libjson-rpc-cpp/archive/v0.7.0.tar.gz"
-  sha256 "669c2259909f11a8c196923a910f9a16a8225ecc14e6c30e2bcb712bab9097eb"
+  url "https://github.com/cinemast/libjson-rpc-cpp/archive/v1.0.0.tar.gz"
+  sha256 "888c10f4be145dfe99e007d5298c90764fb73b58effb2c6a3fc522a5b60a18c6"
   revision 1
```

Comment:
On version updates, if there is a revision, it is always removed. Could you remove this line, please?

Once you do and it runs through the CI again, it should be ready to be merged.
(Thanks for the update PR 😄)

Reply:
I did, thanks for reviewing.